src/gc.cr

```diff
@@ -1,13 +1,76 @@
+# :nodoc:
+fun __crystal_malloc(size : UInt32) : Void*
+  GC.malloc(LibC::SizeT.new(size))
+end
+
+# :nodoc:
+fun __crystal_malloc_atomic(size : UInt32) : Void*
+  GC.malloc(LibC::SizeT.new(size))
+end
+
+# :nodoc:
+fun __crystal_realloc(pointer : Void*, size : UInt32) : Void*
+  GC.realloc(pointer, LibC::SizeT.new(size))
+end
+
+# :nodoc:
+fun __crystal_malloc64(size : UInt64) : Void*
+  {% if flag?(:bits32) %}
+    if size > UInt32::MAX
+      raise ArgumentError.new("Given size is bigger than UInt32::MAX")
+    end
+  {% end %}
+
+  GC.malloc(LibC::SizeT.new(size))
+end
+
+# :nodoc:
+fun __crystal_malloc_atomic64(size : UInt64) : Void*
+  {% if flag?(:bits32) %}
+    if size > UInt32::MAX
+      raise ArgumentError.new("Given size is bigger than UInt32::MAX")
+    end
+  {% end %}
+
+  GC.malloc(LibC::SizeT.new(size))
```

Comment:
Ditto regarding atomic

Reply:
@bcardiff I think this was addressed in latest change, can you check again? Thank you.